other/block_updates_deletes.yaml

```diff
@@ -8,12 +8,12 @@ metadata:
     policies.kyverno.io/subject: RBAC
     policies.kyverno.io/description: >-
       Kubernetes RBAC allows for controls on kinds of resources or those
-      with specific names. This policy restricts updates and deletes to any
-      Service resource that contains the label `protected="true"` unless by
-      a cluster-admin and thus functions as a more granular option to
-      Kubernetes RBAC. 
+      with specific names. But it does not have the type of granularity often
+      required in more complex environments. This sample restricts updates and deletes to any
+      Service resource that contains the label `protected=true` unless by
+      a cluster-admin.
 spec:
-  validationFailureAction: enforce
+  validationFailureAction: audit
```

Comment:
I understand the motivation to use `audit` as the default, but seems like this policy requires an `enforce` to do what it proposes to do.

Reply:
Good point. This one does need `enforce` mode.